pkg/operator/ceph/cluster/cleanup.go

```diff
@@ -162,6 +162,7 @@ func (c *ClusterController) cleanUpJobTemplateSpec(cluster *cephv1.CephCluster,
 			Volumes:            volumes,
 			RestartPolicy:      v1.RestartPolicyOnFailure,
 			PriorityClassName:  cephv1.GetCleanupPriorityClassName(cluster.Spec.PriorityClassNames),
+			SecurityContext:    &v1.PodSecurityContext{},
```

Comment:
If we set the pod `SecurityContext`, we need to text thoroughly that it does not break the container security context that is already set. While if both are set, I understand the container security context will override the pod security context, we must confirm that this does not cause issues with the volumes that are mounted. IIRC the pod security context may only be used for volumes, which could break the volumes. See where we are setting container security context with the `controller.PodSecurityContext()` method. The best way to test this will be on an openshift cluster where the security contexts are necessary. Do you have an openshift cluster for testing?

Reply:
We don't have any upstream jobs to test openshift, unfortunately. I can potentially test it on an openshift cluster, it will just take me a few days to get to it. I'll plan on it next week.

Reply:
> I can potentially test it on an openshift cluster, it will just take me a few days to get to it. I'll plan on it next week.
it would be brilliant, many thanks! I'll wait the results then.

Reply:
The testing looks good in my openshift cluster, I could not see any side effects of setting the pod's securitycontext to empty like this. It also seems low risk enough to go ahead and backport to 1.15.